definition_validator/validator.cc

```diff
@@ -901,6 +907,27 @@ class ValidateWalk {
         }
     }
 
+    core::AutocorrectSuggestion suggestMethodDefinition(const core::GlobalState &gs,
+                                                        const core::Loc classOrModuleDefLoc,
+                                                        const core::MethodRef abstractMethodRef) {
+        auto [_, indentLength] = classOrModuleDefLoc.findStartOfLine(gs);
+        string indent(indentLength, ' ');
+
+        auto methodDefinition = core::lsp::prettyTypeForMethod(gs, abstractMethodRef, nullptr,
+                                                               abstractMethodRef.data(gs)->resultType, nullptr, true);
+
+        vector<string> reindentedMethodDefinitionLines;
+        absl::c_transform(absl::StrSplit(methodDefinition, "\n"), std::back_inserter(reindentedMethodDefinitionLines),
+                          [indent](auto &line) -> string { return absl::StrCat(fmt::format("{}  ", indent), line); });
+
+        auto reindentedMethodDefinition = absl::StrJoin(reindentedMethodDefinitionLines, "\n");
+
+        return core::AutocorrectSuggestion{
+            fmt::format("Define `{}`", abstractMethodRef.show(gs)),
```

Comment:
Given an unimplemented abstract method called `Parent#foo`, this will result in an autocorrect suggestion that reads: "Define `Parent#foo`". Is that copy appropriate?

The only real-world example of this copy that I've seen is from the TypeScript LSP, where the suggestion reads ["Implement X"](https://github.com/microsoft/TypeScript/blob/b1c2e8caadba7722719ead1dca700fbcc307838d/src/compiler/diagnosticMessages.json#L6766-L6773). I preferred "define" because the fix won't generally provide an implementation that passes type checks, and so "implement" felt misleading.

Reply:
"Missing definition" in the error message coupled with "define x" is fine with me.